easybuild/easyconfigs/b/bx-python/bx-python-0.8.2-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'Bundle'
```

Comment:
@SethosII Please consider changing this to `PythonBundle` (& clean up the easyconfig accordingly, see #7124)